mantidproject/mantid
# Update ALFView interface documentation

**Description of work.**
This PR updates the interface documentation for ALFView, and adds release notes for recent feature changes made to the GUI.

The PR also adds a description for how to delete a shape when it is selected.

**To test:**
Build `docs-html`
Open the ALFView documentation page and read it. Make sure it makes sense/all links work/all images are displayed
Read the release notes

Fixes #34824

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
